c/dao.c

```diff
@@ -33,9 +34,13 @@
 #error "Exhausted marker cannot be the same as CKB_INDEX_OUT_OF_BOUND!"
 #endif
 
-// Common definitions here, one important limitation, is that this script only works
-// with scripts and witnesses that are no larger than 32KB. We believe this should be enough
-// for most cases.
+#define CWHR_DEBUG(...)
+#define CWHR_ERROR_CODE -22
+#include "witness_args_handwritten_reader.h"
+
+// Common definitions here, one important limitation, is that this script only
+// works with scripts and witnesses that are no larger than 32KB. We believe
```

Comment:
Witnesses can be removed here?

```suggestion
// works with scripts that are no larger than 32KB. We believe
```

Reply:
The name of the const `MAX_WITNESS_SIZE` is also confusing after the change, consider renaming it to `WITNESS_BUF_SIZE`

Reply:
Already updated